scss/components/_button.scss

```diff
@@ -303,10 +302,57 @@ table.button.warning:hover {
   table {
     td {
       background: darken($warning-color, 10%);
+      border-color: darken($warning-color, 10%);
     }
 
     a {
       border: 0px solid darken($warning-color, 10%);
     }
   }
 }
+
+// Hollow Buttons
+@mixin hollowButton($color,$class:false) {
```

Comment:
Also a little confused by the naming convention - it looks like this is not all of the hollow button styles, but simply the color-specific pieces... maybe we should have 2 mixins? `hollow-button-core` for what is now directly inline, and `hollow-button-color` for this mixin?